json-endpoints.php

```diff
@@ -2493,6 +2493,8 @@
 		'jetpack_testimonial_posts_per_page'   => '(int) Number of testimonials to show per page',
 		'jetpack_portfolio'                    => '(bool) Whether portfolio custom post type is enabled for the site',
 		'jetpack_portfolio_posts_per_page'     => '(int) Number of portfolio projects to show per page',
+		'advanced_seo_front_page_description'  => '(string) The seo meta description for the site.',
```

Comment:
What do you think about giving the options a different name? It'd be nice if they were prefixed with `jetpack_` so folks know where the options come from. `jetpack_seo` sounds descriptive enough to me. It's also a bit more descriptive than `advanced_seo`.